src/api/customer/customer-serializer.ts

```diff
@@ -19,6 +19,72 @@ const createCustomerSerializer: CustomerCreationRequestHandler = (req, res, next
     next();
 }
 
+const paginationSerializer: ListAllCustomersRequestHandler = (req, res, next) => {
+    const PaginationParams = req.query;
+    let offset = 0;
+    let limit = 50;
+  
+    if (PaginationParams.offset) {
+      offset = Number(PaginationParams.offset);
+    }
+  
+    if (PaginationParams.limit) {
+      limit = Number(PaginationParams.limit);
+    }
+  
+    res.locals.paginationParamsSerializer = { offset, limit };
+  
+    next();
+};
+
+const listAllCustomersSerializer: ListAllCustomersRequestHandler = (req, res, next) => {
+    const customersAll = res.locals.listCustomer;
+    res.locals.customerToRespond = [];
+
+    customersAll.map((customers) => {
+        res.locals.customerToRespond.push({
+            uuid: customers.uuid,
+            name: customers.name,
+            contact: {
+                email: customers.email,
+                phone: customers.phone,
+            },
+            document: {
+                cpf: customers.cpf,
+                cnpj: customers.cnpj,
+            },
+            createdAt: customers.createdAt.toISOString(),
+            updatedAt: customers.updatedAt.toISOString(),
+        });
+    });
```

Comment:
Apenas sugestão: desse outro modo simplifica.
Não é necessário criar como um array vazio, e fazer push na sequência.

```suggestion
    res.locals.customerToRespond = customersAll.map((customers) => ({
          uuid: customers.uuid,
          name: customers.name,
          contact: {
              email: customers.email,
              phone: customers.phone,
          },
          document: {
              cpf: customers.cpf,
              cnpj: customers.cnpj,
          },
          createdAt: customers.createdAt.toISOString(),
          updatedAt: customers.updatedAt.toISOString(),
    }));
```